src/commands/push_build_web_assets.yml

```
@@ -0,0 +1,20 @@
+description: >
+  This job sync ui assets with web assets to google storage bucket.
+  Authentication is needed for creating/deleting storage objects.
+parameters:
+  app_name:
+    type: string
+  source:
+    type: string
+    default: ./build/assets
+steps:
+  - attach_workspace:
+      at: .
+  - run:
+      name: Authenticate to GKE
+      command: |
+        echo ${GCLOUD_WEB_ASSETS_KEY} > ${HOME}/gcp-key.json
+        gcloud auth activate-service-account --key-file ${HOME}/gcp-key.json
+  - run:
+      name: Sync build content with bucket
+      command: gsutil -m rsync -r << parameters.source >> gs://${GCLOUD_WEB_ASSETS_BUCKET_NAME}/<< parameters.app_name >>
```

```suggestion
      command: gsutil -m rsync -r << parameters.source >> gs://<< parameters.bucket_name >>
```

Change like this make command more generic. Then when passing `bucket_name` you could use `${GCLOUD_WEB_ASSETS_BUCKET_NAME}/my-app-name`. 

I'm sure this is correct, but you could publish dev version and test it(use published dev version).

---

To simplify usage in the app, I created `bucket_name` parameter with default value ${GCLOUD_WEB_ASSETS_BUCKET_NAME}. This way we don't need to care about environment variable when when we us it in the app.